interface/src/grid.test.js

```diff
@@ -0,0 +1,40 @@
+import React from "react"
+import { shallow, mount } from 'enzyme'
+import Grid from './components/grid'
+
+describe('Test the Grid Component',()=>{
+    it('it should render correctly',()=>{
+        let experiments=[
```

Comment:
Another chance for enhancement: loading from file would be nice to allow easy changing of data sets

Reply:
Similar to the previous one, it also depends on the visualization part

Reply:
here, the goal is to isolate the test of the interface from the test of the server loading the files